recipes/cubicinterpolation/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, CMake, tools
+import glob
+import os
+
+class CubicInterpolationConan(ConanFile):
+    name = "CubicInterpolation"
+    homepage = "https://github.com/MaxSac/cubic_interpolation"
+    version = "0.1"
```

Comment:
```suggestion
```

Do not include version in recipe. Instead, use package reference when building `conan create cubic-interpolation/0.1@`